sys/netbsd/sys.txt

```diff
@@ -190,6 +194,46 @@ ucred {
 	gid	gid
 }
 
+posix_spawn_file_actions {
+	size	int32
+	len	int32
```

Comment:
At least this needs to be `len[fae, int32]`. Not sure what's the difference between size and len. Potentially we need to pass `len[fae, int32]` as size as well.